src/Components/3DV/SceneView.tsx

```diff
@@ -1663,9 +1681,30 @@ function SceneView(props: ISceneViewProps, ref) {
                     barHeight={10}
                 />
             )}
-            {isLoading === undefined && (
-                <div className={customStyles.errorMessage}>
-                    Error loading model. Try Ctrl-F5
+            {loadingError === 'common' && (
+                <div className={customStyles.commonErrorMessage}>
```

Comment:
Is there a reason we dont want to use an illustration message for this as well?  Probably a different illustartion than the CORS error, but might as well use that component since the styling is nice (vs just literal text)

Reply:
yeah, we can :)